package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "doppler-webapp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-scripts": "2.1.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
```

Comment:
last version of IE it's 11, why not just to clarify this subject, just say: "not ie at all"

Reply:
I think that we need to support IE 11, there are IE 11 in Doppler.

Reply:
Yes we need IE 11 sadly.

Reply:
No so sadly, in this project we have a transpiling, so, it is not a big issue.